### PR TITLE
chore: add docs for npd-kdebug

### PR DESCRIPTION
**Pull request by frankqianms**
Add docs to describe the features, deployment, and usages of npd-kdebug.
Still a draft, not ready.

### CODE REVIEW

File: deploy/node-problem-detector/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+* `Event`: Temporary problem that has limited impact on pod but is informative
+should be reported as `Event`.
+
+We call the integration of node-problem-detector(npd) and kdebug as **npd-kdebug**.
+
+## How to deploy npd-kdebug
+
+We have already prepared a [template yaml](./node-problem-detector.yaml) file for you to help you deploy the intergration application of npd-kdebug. You can run the following command to deploy the integrated daemon app to your kubernetes cluster.
+```shell
+kubectl deploy -f ./node-problem-detector/node-problem-detector.yaml
```
[the remark this answers is not in the document]
Comment (frankqianms) [reply deploy/node-problem-detector/README.md]: fixed

File: deploy/node-problem-detector/README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+## What is npd-kdebug
+
+node-problem-detector aims to make various node problems visible to the upstream layers in the cluster management stack. It is a daemon that runs on each node, detects node problems and reports them to apiserver. node-problem-detector can either run as a DaemonSet or run standalone. Now it is running as a Kubernetes Addon enabled by default in the GCE cluster.
+
+In this project, we integrate the node-problem-detector with kdebug. After deploying kdebug with a specific check mode and node-problem-detector, kdebug will try to detect potential problems under the check mode. The results of the check will pass to node-problem-detector, and be reported through problem APIs of node-problem-detector.
```
[the remark this answers is not in the document]
Comment (frankqianms) [reply deploy/node-problem-detector/README.md]: fixed

File: deploy/node-problem-detector/README.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
+## What is npd-kdebug
+
+node-problem-detector aims to make various node problems visible to the upstream layers in the cluster management stack. It is a daemon that runs on each node, detects node problems and reports them to apiserver. node-problem-detector can either run as a DaemonSet or run standalone. Now it is running as a Kubernetes Addon enabled by default in the GCE cluster.
```
[the remark this answers is not in the document]
Comment (frankqianms) [reply deploy/node-problem-detector/README.md]: added

**Review by ArchangelSDY — APPROVED** [review]

**Comment by ArchangelSDY** [comment]
Thanks! I'm merging this. You can open another PR if you have more updates.

**Comment by qianwenyuan** [comment]
> Thanks! I'm merging this. You can open another PR if you have more updates.

Thank you @ArchangelSDY . I will open a new pr for integration of Prometheus.

**Pull request merged**